galera/src/replicator_smm.cpp

```diff
@@ -3009,31 +2862,43 @@ void galera::ReplicatorSMM::process_st_required(
     log_info << "App waits SST: " << app_waits_sst;
 #endif
     // GCache::seqno_reset() happens here
+#ifdef PXC
+    bool error = false;
+    long ret =
+#endif /* PXC */
     request_state_transfer (recv_ctx,
                             group_proto_ver, group_uuid, group_seqno, app_req,
                             app_req_len);
+#ifdef PXC
```

Comment:
Because replicator_smm was refactored by upstream. This is where this part logically should be. While it's at a different location, it's at the same place in the logic of the code.